controller/controller_test.go

```diff
@@ -255,14 +255,15 @@ func newInstanceManager(
 	instanceEngines map[string]longhorn.InstanceProcess,
 	instanceReplicas map[string]longhorn.InstanceProcess,
 	dataEngine longhorn.DataEngineType,
+	instanceManagerImage string,
 	isDeleting bool) *longhorn.InstanceManager {
 
 	im := &longhorn.InstanceManager{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: TestNamespace,
 			UID:       uuid.NewUUID(),
-			Labels:    types.GetInstanceManagerLabels(nodeID, TestInstanceManagerImage, longhorn.InstanceManagerTypeAllInOne, dataEngine),
+			Labels:    types.GetInstanceManagerLabels(nodeID, instanceManagerImage, longhorn.InstanceManagerTypeAllInOne, dataEngine),
```

Comment:
I don't think the result will be different after passing a name instead of using the hard-coded value. It seems you are still passing `TestInstanceManagerImage` for each function call. Or I missed something?

Reply:
Check this one

![image](https://github.com/longhorn/longhorn-manager/assets/12527233/cbbbd1f0-0718-4bc5-baa8-c936e3f7e2c8)